index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" />
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="name">
+            Aleazar Yilma
+        </div>
+        <nav class="navigation">
+            <i class="fa-solid fa-bars"></i>
+        </nav>
+    </header>
+    <main class="content">
+        <section class="title">
+            Aleazar Yilma
+
+        </section>
+        <section class="occupation">
+            Software Developer
+        </section>
+        <section class="intoduction">
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded, don’t hestiate to contact me
+
+            </p>
+        </section>
+        <section class="stacks">
+            <section class="language tech">
+                <span class="stackesHeader"> Language</span>
+                <ul>
+                    <li>Javascript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                </ul>
+            </section>
+            <section class="framework tech">
+                <span class="stackesHeader"> Framework</span>
+                <ul>
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                </ul>
+            </section>
+            <section class="skill tech">
+                <span class="stackesHeader"> Skills</span>
+                <ul>
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+            </section>
+        </section>
+        <section class="socialmedia">
+            <a href="">
+                <img src="/images/icons/github.png" alt="github">
+            </a>
+            <a href="">
+                <img src="/images/icons/linkedin.png" alt="peace">
+            </a>
+            <a href="">
+                <img src="/images/icons/peace.png" alt="peace">
+            </a>
+            <a href="">
+                <img src="/images/icons/twitter.png" alt="peace">
+            </a>
+            <a href="">
+                <img src="/images/icons/m.png" alt="M">
+            </a>
+        </section>
```

Comment:
- [ ] `required` Kindly add this in a `ul` and `li` see screenshot below 
![Screenshot from 2022-06-29 13-54-35](https://user-images.githubusercontent.com/66230306/176757352-df6d738b-f79e-4ca7-b123-1409928a4f5d.png)

- [ ] `required` please also fix the erros see screenshot below

your image
![Screenshot from 2022-06-29 15-04-50](https://user-images.githubusercontent.com/66230306/176758186-2ee2a02b-aad0-4220-9e41-7ea622f5582d.png)

figma template
![Screenshot from 2022-06-29 15-08-46](https://user-images.githubusercontent.com/66230306/176758542-5a6a92ec-cc33-411c-9d4b-1d191c28d666.png)